tools/neat_code_gen/src/CodeGenerator.cpp

```diff
@@ -672,94 +607,65 @@ bool CodeGenerator::reflects_private_members(ifc::DeclIndex index)
 	return false;
 }
 
-bool CodeGenerator::is_type_exported(ifc::TypeIndex type_index)
+bool CodeGenerator::is_type_exported(reflifc::Type type)
 {
-	switch (type_index.sort())
+	switch (type.sort())
 	{
 	case ifc::TypeSort::Fundamental:
 		return true;
 	case ifc::TypeSort::Designated:
-		{
-			const auto& designated_type = file.designated_types()[type_index];
-			return is_type_exported(designated_type.decl);
-		}
-		break;
-	case ifc::TypeSort::Method:
-		{
-			const auto& method = file.method_types()[type_index];
```

Comment:
You removed this switch case here, but thinking about it, it should always be exported if I add forward declarations to NeatReflection.